control/controller.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, connection_string, baud=None):
         self.logger = logging.getLogger(__name__)
         self.connection_string = connection_string
         self.vehicle = None
+        self.__com = com
         if baud:
```

Comment:
I thought we had some kind of Connecting... and Connected message using the com around here, did we not?